### ISSUE TITLE
Unearthing demo not visible in publiclab.org

### ISSUE THREAD

**Issue by jywarren** [publiclab/leaflet-environmental-layers#349]
Hi all, i'm not sure what happened, as we haven't published anything i think... but I don't see the markers on https://publiclab.org/wiki/unearthing-pvd any more. @sagarpreet-chadha @crisner @nstjean can we try to prioritize debugging this? 

Thank you -- i guess we need a system test in plots2 to ensure this stays live? There are a lot of moving parts!

**Comment by jywarren** [comment]
![image](https://user-images.githubusercontent.com/24359/72481992-2f9d9780-37ca-11ea-90e2-50071b3c475d.png)

```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.assign (<anonymous>)
    at NewClass.onAdd (LeafletEnvironmentalLayers.js:25977)
    at NewClass._layerAdd (LeafletEnvironmentalLayers.js:18527)
    at NewClass.whenReady (LeafletEnvironmentalLayers.js:16407)
    at NewClass.addLayer (LeafletEnvironmentalLayers.js:18589)
    at NewClass.addTo (LeafletEnvironmentalLayers.js:18469)
    at setupInlineLEL (xml.js:30)
    at unearthing-pvd:1067
```

**Comment by crisner** [comment]
Working on this.

**Comment by crisner** [comment]
So, the error from the console has been fixed in PR #351 under a new issue #350 as this wasn't responsible for the markers not showing up. 

It looks like the layers are available on the map but not added to the map by default.
![publiclab org_wiki_unearthing-pvd (1)](https://user-images.githubusercontent.com/29401459/72509783-4ac1e300-386e-11ea-995d-682723803fc0.png)

When I was working on the embed feature I removed the code responsible for this here: https://github.com/publiclab/leaflet-environmental-layers/blame/00c10f2c157a90a1d98a0254a061f9d6dd8f4c57/src/AllLayers.js#L123-L125,
 because it was adding all the layers by default and I hadn't realized its usage in the inline maps section. I shall add a fix where if layers are passed in the `include` option then those layers would be added to the map by default.
